### ISSUE TITLE
a11y: Expose accessibility strings in shortcuts tree

### ISSUE THREAD

**Issue by jcsteh** [mumble-voip/mumble#2293]
The shortcuts tree accessible via Configure menu -> Settings -> Shortcuts is very inaccessible for screen reader users. The big issue is that the items in the tree don't expose any usable accessibility information. The "Function" column seems to expose a number based on the action (e.g. "1" is exposed for Push-to-Talk), which isn't meaningful to users. The other columns don't expose anything at all. It would be great if all of the items exposed a name for accessibility indicating their value.

I believe this can be done using the [Qt::AccessibleTextRole](http://doc.qt.io/qt-5/qt.html#ItemDataRole-enum).

Another issue is that you often can't navigate through the rows of this tree with the up and down arrow keys. These instead get swallowed by the pop-up inside the tree item. You also can't navigate by column. However, I imagine these may be issues that require fixes in QT itself.

**Comment by mkrautz** [comment]
- [ ] Add accessibility information to the shortcuts tree (Qt::AccessibleTextRole)
- [ ] Make it possible to navigate the tree with the keyboard (columns and rows, shortcut popup, data popup, set shortcut, etc.)

**Comment by dingpengyu** [comment]
cc @jcsteh 
Is this problem solved?
thanks

**Comment by jcsteh** [comment]
Unfortunately, I no longer use Mumble and thus no longer have it set up. I
don't anticipate having the time to test this. Sorry I'm unable to help. :(
